GVRf/Framework/jni/engine/renderer/renderer.cpp

```diff
@@ -504,66 +502,80 @@ void Renderer::renderRenderData(RenderData* render_data,
-            glm::mat4 mvp_matrix(projection_matrix * mv_matrix);
-            try {
-                bool right = render_mask & RenderData::RenderMaskBit::Right;
-                switch (render_data->material()->shader_type()) {
-                case Material::ShaderType::UNLIT_SHADER:
-                    shader_manager->getUnlitShader()->render(mvp_matrix,
-                            render_data);
-                    break;
-                case Material::ShaderType::UNLIT_HORIZONTAL_STEREO_SHADER:
-                    shader_manager->getUnlitHorizontalStereoShader()->render(
-                            mvp_matrix, render_data, right);
-                    break;
-                case Material::ShaderType::UNLIT_VERTICAL_STEREO_SHADER:
-                    shader_manager->getUnlitVerticalStereoShader()->render(
-                            mvp_matrix, render_data, right);
-                    break;
-                case Material::ShaderType::OES_SHADER:
-                    shader_manager->getOESShader()->render(mvp_matrix,
-                            render_data);
-                    break;
-                case Material::ShaderType::OES_HORIZONTAL_STEREO_SHADER:
-                    shader_manager->getOESHorizontalStereoShader()->render(
-                            mvp_matrix, render_data, right);
-                    break;
-                case Material::ShaderType::OES_VERTICAL_STEREO_SHADER:
-                    shader_manager->getOESVerticalStereoShader()->render(
-                            mvp_matrix, render_data, right);
-                    break;
-                case Material::ShaderType::CUBEMAP_SHADER:
-                    shader_manager->getCubemapShader()->render(model_matrix,
-                            mvp_matrix, render_data);
-                    break;
-                case Material::ShaderType::CUBEMAP_REFLECTION_SHADER:
-                    shader_manager->getCubemapReflectionShader()->render(
-                            mv_matrix, glm::inverseTranspose(mv_matrix),
-                            glm::inverse(view_matrix), mvp_matrix, render_data);
-                    break;
-                default:
-                    shader_manager->getCustomShader(
-                            render_data->material()->shader_type())->render(
-                            mvp_matrix, render_data, right);
-                    break;
-                }
-            } catch (std::string error) {
-                LOGE(
-                        "Error detected in Renderer::renderRenderData; name : %s, error : %s",
-                        render_data->owner_object()->name().c_str(),
-                        error.c_str());
-                shader_manager->getErrorShader()->render(mvp_matrix,
-                        render_data);
+            for (int curr_pass = 0; curr_pass < render_data->pass_count(); ++curr_pass) {
+            	numberTriangles += render_data->mesh()->getNumTriangles();
+				numberDrawCalls++;
```

Comment:
take a look at the diff here on github.  you'll see this line as well as all the rest below it are indented like 16 characters.  it looks like you fixed the indentation on the previous line, now just fix it for the rest :)